2020-06-02 15:17:50,731 - app.py - <module> - 11 - INFO - Now Starting!
2020-06-02 15:17:50,749 - _internal.py - _log - 113 - INFO -  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
2020-06-02 15:38:23,150 - app.py - <module> - 11 - INFO - Now Starting!
2020-06-02 15:38:23,166 - _internal.py - _log - 113 - INFO -  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
2020-06-02 15:38:28,072 - _internal.py - _log - 113 - INFO - 127.0.0.1 - - [02/Jun/2020 15:38:28] "[37mGET / HTTP/1.1[0m" 200 -
2020-06-02 21:49:18,531 - app.py - <module> - 11 - INFO - Now Starting!
2020-06-02 21:49:18,548 - _internal.py - _log - 113 - INFO -  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
2020-06-02 21:49:22,382 - _internal.py - _log - 113 - INFO - 127.0.0.1 - - [02/Jun/2020 21:49:22] "[37mGET / HTTP/1.1[0m" 200 -
2020-06-02 21:49:33,822 - route.py - GetCourseInfoStu - 101 - INFO - Get : 161830317
2020-06-02 21:49:33,822 - app.py - log_exception - 1891 - ERROR - Exception on /classpoint/getcourseinfo/stu [GET]
Traceback (most recent call last):
  File "C:\Users\xjhznb\PycharmProjects\untitled1\venv\lib\site-packages\flask\app.py", line 2446, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Users\xjhznb\PycharmProjects\untitled1\venv\lib\site-packages\flask\app.py", line 1951, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Users\xjhznb\PycharmProjects\untitled1\venv\lib\site-packages\flask\app.py", line 1820, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "C:\Users\xjhznb\PycharmProjects\untitled1\venv\lib\site-packages\flask\_compat.py", line 39, in reraise
    raise value
  File "C:\Users\xjhznb\PycharmProjects\untitled1\venv\lib\site-packages\flask\app.py", line 1949, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Users\xjhznb\PycharmProjects\untitled1\venv\lib\site-packages\flask\app.py", line 1935, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "C:\Users\xjhznb\PycharmProjects\untitled1\api_1_0_0\route.py", line 102, in GetCourseInfoStu
    status, current, _next = method.GetCourseInfoStu(stu_id)
TypeError: 'NoneType' object is not iterable
2020-06-02 21:49:33,853 - _internal.py - _log - 113 - INFO - 127.0.0.1 - - [02/Jun/2020 21:49:33] "[35m[1mGET /classpoint/getcourseinfo/stu?stu_id=161830317 HTTP/1.1[0m" 500 -
2020-06-02 21:49:43,696 - _internal.py - _log - 113 - INFO - 127.0.0.1 - - [02/Jun/2020 21:49:43] "[37mGET /log HTTP/1.1[0m" 200 -
2020-06-02 21:50:13,510 - app.py - <module> - 11 - INFO - Now Starting!
2020-06-02 21:50:13,527 - _internal.py - _log - 113 - INFO -  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
2020-06-02 21:50:17,021 - route.py - GetCourseInfoStu - 101 - INFO - Get : 161830317
2020-06-02 21:50:17,214 - route.py - GetCourseInfoStu - 104 - INFO - Return : {'status': 0, 'current': {'have_classes_or_not': False}, 'next': {'have_classes_or_not': True, 'course_info': {'base_course_id': '5ed4e01b32bfbd294094c903', 'dyn_course_id': '5ed5fb73f52b7257881d82f3', 'course_name': 'Êý¾Ý½á¹¹', 'lecturer_name': 'gh', 'campus': 1, 'classroom': 'D1203', 'main_class': ['1618303'], 'week': 1, 'class_time': datetime.datetime(2020, 6, 2, 16, 15), 'sign_in_status': 0}}}
2020-06-02 21:50:17,215 - _internal.py - _log - 113 - INFO - 127.0.0.1 - - [02/Jun/2020 21:50:17] "[37mGET /classpoint/getcourseinfo/stu?stu_id=161830317 HTTP/1.1[0m" 200 -
2020-06-02 21:50:22,052 - _internal.py - _log - 113 - INFO - 127.0.0.1 - - [02/Jun/2020 21:50:22] "[37mGET /log HTTP/1.1[0m" 200 -
2020-06-02 21:50:43,266 - route.py - GetCourseInfoStu - 101 - INFO - Get : 161830317
2020-06-02 21:50:43,300 - route.py - GetCourseInfoStu - 104 - INFO - Return : {'status': 0, 'current': {'have_classes_or_not': False}, 'next': {'have_classes_or_not': True, 'course_info': {'base_course_id': '5ed4e01b32bfbd294094c903', 'dyn_course_id': '5ed5fb73f52b7257881d82f3', 'course_name': 'Êý¾Ý½á¹¹', 'lecturer_name': 'gh', 'campus': 1, 'classroom': 'D1203', 'main_class': ['1618303'], 'week': 1, 'class_time': datetime.datetime(2020, 6, 2, 16, 15), 'sign_in_status': 0}}}
2020-06-02 21:50:43,300 - _internal.py - _log - 113 - INFO - 127.0.0.1 - - [02/Jun/2020 21:50:43] "[37mGET /classpoint/getcourseinfo/stu?stu_id=161830317 HTTP/1.1[0m" 200 -
2020-06-02 21:52:32,144 - app.py - <module> - 11 - INFO - Now Starting!
2020-06-02 21:52:32,161 - _internal.py - _log - 113 - INFO -  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
2020-06-02 21:52:34,161 - route.py - GetCourseInfoStu - 101 - INFO - Get : 161830317
2020-06-02 21:52:34,373 - route.py - GetCourseInfoStu - 104 - INFO - Return : {'status': 1, 'current': None, 'next': None}
2020-06-02 21:52:34,374 - _internal.py - _log - 113 - INFO - 127.0.0.1 - - [02/Jun/2020 21:52:34] "[37mGET /classpoint/getcourseinfo/stu?stu_id=161830317 HTTP/1.1[0m" 200 -
2020-06-02 21:52:50,373 - app.py - <module> - 11 - INFO - Now Starting!
2020-06-02 21:52:50,390 - _internal.py - _log - 113 - INFO -  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
2020-06-02 21:52:52,986 - route.py - GetCourseInfoStu - 101 - INFO - Get : 161830317
2020-06-02 21:52:53,056 - route.py - GetCourseInfoStu - 104 - INFO - Return : {'status': 1, 'current': None, 'next': None}
2020-06-02 21:52:53,057 - _internal.py - _log - 113 - INFO - 127.0.0.1 - - [02/Jun/2020 21:52:53] "[37mGET /classpoint/getcourseinfo/stu?stu_id=161830317 HTTP/1.1[0m" 200 -
